docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1729,6 +1729,16 @@ The limit protects the system’s stability from potential abuse or mistakes. To
 Invalid series are skipped during the ingestion, and valid series within the same request are ingested.
 {{< /admonition >}}
 
+### err-mimir-max-label-names-per-info-series
+
+This non-critical error occurs when Mimir receives a write request that contains an info series with a number of labels that exceed the configured limit.
+An info series is a series where the metric name ends in `_info`.
+The limit protects the system’s stability from potential abuse or mistakes. To configure the limit on a per-tenant basis, use the `-validation.max-label-names-per-info-series` option.
```

Comment:
To configure per tenant, you need YAML don't you rather than the flag:
```suggestion
The limit protects the system’s stability from potential abuse or mistakes. To configure the limit on a per-tenant basis, use the `validation.max_label_names_per_info_series` option.
```

Reply:
ack, I'll update this and where I copy pasted from above as well

Reply:
also it becomes `limits.max_label_names_per_info_series` I think

Reply:
Actually there's 10 places where we use `-validation...` flag and zero where we write yaml option. Do you want to change all?

Reply:
I certainly wouldn't change existing code as part of this PR. I just don't think it makes sense to refer to a flag, when you have to use YAML per-tenant.